### PR TITLE
Cairographics family: add license

### PR DESCRIPTION
**Pull request by miccal**
Helping out on https://github.com/Homebrew/homebrew-core/issues/58225

### CODE REVIEW

**Comment by miccal** [comment]
Ping @SeekingMeaning @chenrui333 @Rylan12

File: Formula/cairomm.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class Cairomm < Formula
   homepage "https://cairographics.org/cairomm/"
   url "https://cairographics.org/releases/cairomm-1.12.2.tar.gz"
   sha256 "45c47fd4d0aa77464a75cdca011143fea3ef795c4753f6e860057da5fb8bd599"
+  license "LGPL-2.1"
```
Comment (Rylan12) [inline Formula/cairomm.rb:6]: I just want to clarify: LGPL-2.1 refers to the "GNU Lesser General Public License", correct?

On the [homepage](https://cairographics.org/cairomm/) for `cairomm` it says:

> cairomm is free software; you can redistribute it and/or modify it under the terms of the GNU Library General Public License as published by the Free Software Foundation; either version 2 of the License, or (at your option) any later version.
>
> This library is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the [GNU Library General Public License](http://www.gnu.org/licenses/lgpl.html) for more details.

This indicates that it uses the "GNU Library General Public License" (confusingly also LGPL)

To add further confusion, the homepage links to the [Lesser GPL](https://www.gnu.org/licenses/lgpl-3.0.html) license page

Also, the main page for the [Library GPL](https://www.gnu.org/licenses/old-licenses/lgpl-2.0.en.html) license mentions that it has been superseded by the [Lesser GPL](https://www.gnu.org/licenses/lgpl-3.0.html) license.

Bottom line: are Lesser GPL and Library GPL effectively the same thing? Is it okay to say LGPL and refer to Lesser GPL in some places and Library GPL in others? Part of me thinks yes because one license supersedes the other, but part of me thinks no because they are different licenses and we should be clear.

Comment (miccal) [reply Formula/cairomm.rb:6]: I just went with what it said on `cairographics.org`, namely:

> Cairo is free software and is available to be redistributed and/or modified under the terms of either the GNU Lesser General Public License (LGPL) version 2.1 or the Mozilla Public License (MPL) version 1.1 at your option.

I (perhaps mistakenly) assumed this applied to all that was downloaded from there?

Comment (Rylan12) [reply Formula/cairomm.rb:6]: No worries! I genuinely don't know the answer... maybe @chenrui333 can weigh in?

On the [LGPL Wikipedia page](https://en.wikipedia.org/wiki/GNU_Lesser_General_Public_License), it says that the Lesser GPL license was originally called the Library GPL license, so I think they're probably the same thing and so LPGL is an acceptable abbreviation

Comment (SMillerDev) [reply Formula/cairomm.rb:6]: This seems like a good question to ask upstream (which one they mean), but they have different identifiers anyway: https://opensource.org/licenses/lgpl-license

Comment (SMillerDev) [reply Formula/cairomm.rb:6]: We should use the defined license. While it would be allowed it's rarely done this way, as explained [for GPL](https://opensource.org/licenses/gpl-license) as well

Comment (Rylan12) [reply Formula/cairomm.rb:6]: Okay, so it sounds like the best option is just to change to LGPL-2.0 for `cairomm`

Comment (miccal) [reply Formula/cairomm.rb]: Done.

**Review by Rylan12 — APPROVED** [review]

**Comment by Rylan12** [comment]
Thanks, @miccal!

**Comment by miccal** [comment]
You're welcome @Rylan12
